app/views/home/index.html.erb

```diff
@@ -0,0 +1,18 @@
+<div>
+  <h1>Inimitable</h1>
+  <div>
+    <h2>Your Poems</h2>
+    <% if @poems.present? %>
+      <dl>
+        <% @poems.each do |poem| %>
+          <dt><%= poem.title %></dt>
+          <dd><%= poem.status %></dd>
+          <dd><%= poem.updated_at %></dd>
+        <% end %>
+      </dl>
+    <% else %>
+      <p>Looks like you don't have any poems yet.</p>
+    <% end %>
+    <%= button_to t("poems.actions.new"), new_poem_path, method: :get %>
```

Comment:
I was trying to focus on wiring up the functionality and not worry about applying any css classes until I had wireframes together. Certainly make sense to stick with `link_to` for `get` and `button_to` for `post` instead of overriding the method.